Diagnostics/HealthChecker/Analyzer/Security/Invoke-AnalyzerSecurityExchangeCertificates.ps1

```diff
@@ -130,6 +133,15 @@ function Invoke-AnalyzerSecurityExchangeCertificates {
             Add-AnalyzedResultInformation @params
         }
 
+        if ($null -ne $certificate.IsEccCertificate) {
```

Comment:
`$certificate.IsEccCertificate` appears to be a `bool` so if set to `false`, this will still display. What are the scenarios where this is going to be `null`?

Reply:
Right. The null-value check was a leftover from initial testing. Will be removed with the next commit.